.github/workflows/test-lightwalletd.yml

```diff
@@ -0,0 +1,319 @@
+name: Test lightwalletd
+
+on:
+  workflow_dispatch:
+    inputs:
+      network:
+        default: 'Mainnet'
+        description: 'Network to deploy: Mainnet or Testnet'
+        required: true
+      regenerate-disks:
+        type: boolean
+        default: false
+        description: 'Just update lightwalletd stateful disks'
+        required: true
+  pull_request:
+    branches:
+      - main
+    paths:
+      # code and tests
+      - '**/*.rs'
+      # hard-coded checkpoints and proptest regressions
+      - '**/*.txt'
+      # test data snapshots
+      - '**/*.snap'
+      # dependencies
+      - '**/Cargo.toml'
+      - '**/Cargo.lock'
+      # workflow definitions
+      - 'docker/**'
+      - '.github/workflows/test-lightwalletd.yml'
+
+  push:
+    branches:
+      - main
+    paths:
+      # code and tests
+      - '**/*.rs'
+      # hard-coded checkpoints and proptest regressions
+      - '**/*.txt'
+      # test data snapshots
+      - '**/*.snap'
+      # dependencies
+      - '**/Cargo.toml'
+      - '**/Cargo.lock'
+      # workflow definitions
+      - 'docker/**'
+      - '.github/workflows/test-lightwalletd.yml'
+
+env:
+  CARGO_INCREMENTAL: '1'
+  ZEBRA_SKIP_IPV6_TESTS: '1'
+  ZEBRA_TEST_LIGHTWALLETD: '1'
+  RUST_BACKTRACE: full
+  RUST_LIB_BACKTRACE: full
+  COLORBT_SHOW_HIDDEN: '1'
+  NETWORK: Mainnet
+  PROJECT_ID: zealous-zebra
+  GAR_BASE: us-docker.pkg.dev/zealous-zebra/zebra
+  GCR_BASE: gcr.io/zealous-zebra
+  REGION: us-central1
+  ZONE: us-central1-a
+  MACHINE_TYPE: c2d-standard-4
+  IMAGE_NAME: lightwalletd-test
+
+jobs:
+  build:
+    name: Build images
```

Comment:
Can this be replaced by a call to `docker-image-build.yml`?

Reply:
Fixed by [refactor(ci): use workflow_call for build](https://github.com/ZcashFoundation/zebra/pull/4223/commits/e66ac6ef843945b42ff35b71bc677728d2ba32b1)